west.yml

```diff
@@ -259,7 +259,7 @@ manifest:
         - hal
     - name: hostap
       path: modules/lib/hostap
-      revision: a90df86d7c596a5367ff70c2b50c7f599e6636f3
+      revision: pull/23/head
```

Comment:
You need to fix the revision and put here the correct SHA from hostap module that has your change.

Reply:
yes, but I already in another PR update the latest SHA of hostap:
https://github.com/zephyrproject-rtos/zephyr/pull/76824
so I wanted to close this PR as it's not needed anymore,  am I wrong?

Reply:
Sure, if the manifest is already updated, this can be closed.